content/running-amazon-ec2-workloads-at-scale/spot_resilience.md

```diff
@@ -4,7 +4,7 @@ weight = 155
 +++
```

Comment:
A comment that we didn't add to previous code reviews: The title of this section seems a bit strange, "Resilience with Spot instances", lacks something.  'Increasing Spot instances resilience' or similar perhaps is a better option

Reply:
Agree. Changed this with "Increasing resilience when using Spot Instances"